### ISSUE TITLE
failing with Boot 2.6.0

### ISSUE THREAD

**Issue by daslu** [martinklepsch/tenzing#71]
This seems very similar to [#54 of mount](https://github.com/tolitius/mount/issues/54) .
Downgrading to Boot 2.5.5 (through boot.properties) makes the error disappear.

```
lein new tenzing proj1
cd proj1
boot dev
```
results in the following error:
```
...
     clojure.lang.ExceptionInfo: template already refers to: #'boot.core/template in namespace: adzerk.boot-reload
...
```

**Comment by martinklepsch** [comment]
This is fixed in master by upgrading boot reload. Need to cut a release :)

**Comment by martinklepsch** [comment]
Fixed with `0.4.0`
